neural_compressor/compression/pruner/patterns/base.py

```diff
@@ -20,6 +20,8 @@
 
 import numpy as np
 
+from neural_compressor.compression.pruner.utils import safe_get_data, safe_get_grad, safe_get_shape
```

Comment:
for better compatibility of the pruning module on ITREX, it's preferable to replace the import path containing 'neural_compressor'
such as:
from ..utils import safe_get_data, safe_get_grad, safe_get_shape

Reply:
nice catch, updated in [ed6395b](https://github.com/intel/neural-compressor/pull/1492/commits/ed6395b2fa9ae9aecce68b2a0ceb6e3e1ec2f02d).